src/app.ts

```diff
@@ -21,17 +19,21 @@ import https from 'https';
 import WebSocket from 'ws';
 import config from 'config';
 
-if (process.env.NODE_ENV === 'development') {
-    const httpsAgent = new https.Agent({
-        rejectUnauthorized: false,
-    });
+// Manager
+import processEnvManager from './manager/processEnv/processEnv.manager';
+
+// To discuss
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const swaggerDocument = require('./swagger-output.json');
```

Comment:
I suppose the problem is that u're trying to use a json importing it as a module. If not let's discuss what is the problem @Zenomante